www/src/pages/table/NodeSearchComponent.tsx

```diff
@@ -128,19 +128,13 @@ export const NodeSearchComponent = withStyles(styles)(
         classStore
           .fetchSearchResults(query)
           .then((data) => {
-            data.forEach((searchResult: ClassTextSearchResult) => {
+            data.forEach((searchResult: TextSearchResult) => {
               searchResults.push({
                 type: "class",
                 id: searchResult.id,
                 label: (
-                  <div>
-                    <p style={{ color: "blue" }}>
-                      {searchResult.label} ({searchResult.id})
-                    </p>
-                    <p style={{ fontSize: 12, marginTop: -5 }}>
-                      {searchResult.description}
-                    </p>
-                  </div>
+                  <LabelComponent id={searchResult.id} label={searchResult.label}
```

Comment:
Since we don't display uri in the label, we can remove it in the Label Component